webapp-django/crashstats/status/context_processors.py

```diff
@@ -0,0 +1,18 @@
+from crashstats.status.models import StatusMessage
+
+
+def status_message(request):
+    statuses = (
+        StatusMessage.objects.filter(enabled=True).order_by('-created_at')
+    )
+
+    if not len(statuses):
```

Comment:
Why would that not be applicable here? Does it not take filters into consideration?